pastepwn/analyzers/databasedumpanalyzer.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from .basicanalyzer import BasicAnalyzer
+
+
+class DatabaseDumpAnalyzer(BasicAnalyzer):
+    """Analyzer to match database dump"""
+    name = "DatabaseDumpAnalyzer"
+
+    def __init__(self, actions):
+        """
+        Analyzer to match database dump
+        :param actions: A single action or a list of actions to be executed on every paste
+        """
+        # This regex match the columns of a database
+        regex = r"[(]((`\w+`|\d), )+(`\w+`|\d)[)]"
```

Comment:
Nice regex. Two suggestions though: 

1) Instead of using `[(]` you could simply use `\(` - this makes it easier readable.
2) I would replace every literal space with `(\s?)+` - that way you circumvent any issues with whitespaces.

The finished regex could look like this: ```\(((`\w+`|\d)(\s?)+,(\s?)+)+(`\w+`|\d)\)```

Reply:
Smart corrections, I changed the regex